webapp/TargetedMS/js/QCSummaryPanel.js

```diff
@@ -308,14 +310,16 @@ Ext4.define('LABKEY.targetedms.QCSummary', {
                 html += '<tr id="' + sampleFile.calloutId + '"><td><div class="sample-file-item">'
                         + '<span class="fa ' + iconCls + '"></span> ' + Ext4.util.Format.htmlEncode(sampleFile.ReplicateName) + '</div></td><td><div class="sample-file-item-acquired">' + Ext4.util.Format.date(sampleFile.AcquiredTime ? new Date(sampleFile.AcquiredTime) : null, LABKEY.extDefaultDateTimeFormat || 'Y-m-d H:i:s') + '</div></td>';
 
-                html += '<td style="text-align: right"><div class="sample-file-item-outliers">';
                 if (sampleFile.IgnoreForAllMetric) {
-                    html += 'not included in QC';
+                    html += '<td colspan="4"><div class="sample-file-item-outliers" style="text-align: center">excluded</div></td>';
                 }
                 else {
-                    html += totalOutliers;
+                    html += '<td style="text-align: right"><div class="sample-file-item-outliers">' + sampleFile.LeveyJennings + "</div></td>"
```

Comment:
do we need Ext4.util.Format.htmlEncode for values here?

Reply:
Always a good question to ask. :) 

These are integer values so they're safe.